### PR TITLE
selenium support

### PR DESCRIPTION
**Pull request by KevinRodriguez777**

### CODE REVIEW

**Comment by KevinRodriguez777** [comment]
fixes https://github.com/Nuitka/Nuitka/issues/2746

File: .gitignore [hunk 1]
```diff
@@ -67,8 +67,6 @@ doxygen-*
 .coverage*
 *.tmp
 *.dat
-.vscode/ipch
-.vscode/tags
-.vscode/.ropeproject
 Nuitka-*
 nuitka/tools/podman/containers/requirements-devel.txt
+.vscode/
```
Comment (kayhayen) [inline .gitignore:72]: I want to know about files there, that are to be ignored, so as to not missing things not to ignore, this counteracts it, so I won't merge this part.

**Review by kayhayen — APPROVED** [review]

**Comment by kayhayen** [comment]
I am merging this, but I will remove the gitignore immediately, and put it on factory and hotfix branch, so I can have it for 2.1.2 release.

File: .gitignore (hunk 1 above)
Comment (KevinRodriguez777) [reply .gitignore:72]: so you'd rather gitignore specific files?

Comment (kayhayen) [reply .gitignore:72]: Yes, already doing that, I think you removed some, those ones are vscode state files, but every now and then, an extension creates a config file, and that I would hate to accidentally ignore. So whatever brought you to ignoring that, it's worth considering if that file is for git to contain or not on a merits base.

**Comment by furjac** [comment]
Is selenium working now

**Pull request merged**